### DISCUSSION TITLE
"trackables" icon ideas?

### DISCUSSION THREAD

**Discussion by ztNFny** [cgeo/cgeo#12047] [General]
In lists we show the number of trackables currently logged into a cache. I'm looking for an icon for that.

![a](https://user-images.githubusercontent.com/1258173/140342202-46ad50a5-7600-49b6-a51e-4d9366b63200.png)
The old icon: ![image](https://user-images.githubusercontent.com/1258173/140344165-1b2fe88c-a3b8-45a3-9adb-791c349ff073.png)

A generic "trackable" icon that covers all 3 possible sources is required:
- GC Travel Bug or Geocoin
- GeoKrety
- GeoLutins

I wouldn't know of any icon which clearly says "trackable". As of those services GC TBs are (by far) the most popular I'd go into that direction with a "bug" icon that is different from the GC TB but (hopefully) close enough to be recognized as such. Or the dog tag approach with just a tag, but the potential for that to be interpreted as something else seems higher.

But maybe someone has a good idea?

I have the following proposals and would suggest the "usual" emoji as vote approach:
Thumbs up: ![bug](https://user-images.githubusercontent.com/1258173/140340147-919d8302-6dfd-44d8-a0bc-888eeb5a1f82.png)
Thumbs down: ![tag](https://user-images.githubusercontent.com/1258173/140340164-50c68b87-9343-4e9f-96c5-bc673d88e037.png)
Laugh: ![tag-text](https://user-images.githubusercontent.com/1258173/140340155-0ee9d8a3-0eeb-4165-8e7c-f60e158ea3bd.png)
Party: ![baseline_inventory_black_24dp](https://user-images.githubusercontent.com/1258173/140340131-0abe4e42-ad1c-43f0-9a32-589e5fd3211f.png)

**Comment by bekuno** [comment]
By the way, the gray icon on black background is bad visible.
![Screenshot_20211104-181452_cgeo](https://user-images.githubusercontent.com/32555742/140387971-b3213e40-b31b-4f19-b952-3af3b8f578c7.jpg)

**Reply by ztNFny** [reply]
should probably use primaryColor tint - white on black theme, black in white theme

**Reply by ztNFny** [reply]
fixed (in my dev branch)

**Comment by Lineflyer** [comment]
Voted and additional comment:
In fact that icon only shows GC trackables at this moment due to the following facts/shortcomings:
- Geokrety are not fetched while storing the cache and added to the DB but only loaded on runtime
- GeoLutins are not at all supported in connection with caches by c:geo

Still it should stay a generic icon as the missing GeoKrety are more a bug / missing feature
